Doc/library/pathlib.rst

```diff
@@ -1229,6 +1229,56 @@ call fails (for example because the path doesn't exist).
    .. versionchanged:: 3.10
       The *newline* parameter was added.
 
+Subclassing and Extensibility
+-----------------------------
+
+Both :class:`PurePath` and :class:`Path` are directly subclassable and extensible as you
+see fit:
+
+   >>> class MyPath(Path):
+   ...    def my_method(self, *args, **kwargs):
+   ...        ... # Platform agnostic implementation
+
+.. note::
+   Unlike :class:`PurePath` or :class:`Path`, instantiating the derived
+   class will not generate a differently named class:
+
+   .. doctest::
+      :pyversion: > 3.11
+      :skipif: is_windows
+
+      >>> Path('.')  # On POSIX
+      PosixPath('.')
+      >>> MyPath('.')
+      MyPath('.')
+
+   Despite this, the subclass will otherwise match the class that would be
+   returned by the factory on your particular system type. For instance,
+   when instantiated on a POSIX system:
+
+   .. doctest::
+      :pyversion: > 3.11
+      :skipif: is_windows
+
+      >>> [Path('/dir').is_absolute, MyPath('/dir').is_absolute()]
+      [True, True]
+      >>> [Path().home().drive, MyPath().home().drive]
+      ['', '']
```

Comment:
A minor point: the lists here are a little bit distracting (we are looking at methods/properties, not working with lists).
Could use separate lines, or `>>> Path().home().drive, MyPath().home().drive` (which will have parens in the output but that’s fine, we’re used to commas being sometimes multiple things, sometimes a tuple object).